### PR TITLE
module-bridge: Bump SDK version

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: module-bridge/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 //! Bridge runtime module.
+#![deny(rust_2018_idioms)]
```
Comment (pro-wh) [inline module-bridge/src/lib.rs:2]: what does this do

**Review by pro-wh — APPROVED** [review]
I think we were in a rush to get the new artifacts that (1) we switched to GitHub from Buildkite and (2) stopped labeling downloaded artifacts with version info. This PR replicates what we've done in the SDK repo, which is fine for now. I should open an issue in oasis-sdk to list this technical debt.

File: module-bridge/src/lib.rs (hunk 1 above)
Comment (pro-wh) [reply module-bridge/src/lib.rs:2]: oh wait deny means something else :laughing:

Comment (kostko) [reply module-bridge/src/lib.rs:2]: Yeah it reads weird, but deny means that these lints will trigger errors not that we don't want rust-2018-idioms :-)

**Pull request merged**